Resources/views/Block/default.html.twig

```diff
@@ -1,3 +1,5 @@
+{% if content is not empty %}
```

Comment:
This `if` replace this one : https://github.com/M6Web/DraftjsBundle/pull/7/files#diff-e3f77770a594b787a2557a29ce21c748L59